src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,215 @@
+@import '../../variables' ;
+
+.footer {
+    flex-shrink: 0;
+    display: block;
+}
+
+.pt-5 {
+    padding-top: 3rem !important;
```

Comment:
if padding is 3rem then why name is pt-5?

Reply:
@akashsinha13, modified this at many places with proper number